source/val/validate_decorations.cpp

```diff
@@ -848,32 +848,35 @@ spv_result_t CheckDecorationsOfEntryPoints(ValidationState_t& vstate) {
           if (storage_class == spv::StorageClass::TaskPayloadWorkgroupEXT) {
             if (has_task_payload) {
               return vstate.diag(SPV_ERROR_INVALID_ID, var_instr)
-                     << "There can be at most one OpVariable with storage "
+                     << "In SPIR-V 1.4 or later, there can be at most one "
```

Comment:
This also feels misleading. The SPIR-V 1.4 restriction is that all statically used global variables must be mentioned in the entry point interface list. It is incidental that the extension requires SPIR-V 1.4 I think.

If the binary version is less than 1.4, users will get an error that only input or output can be used. In 1.4 or later it should just be a uniqueness error.